tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -20,15 +20,26 @@ JupyterHub Spawner Is Visible
 Select Notebook Image
    [Documentation]  Selects a notebook image based on a partial match of ${notebook_image} argument
    [Arguments]  ${notebook_image}
-   Wait Until Element Is Visible  xpath:/html/body/div[1]/form/div/div/div[2]/div[2]/div[1]
+   ${version-check} =  Is RHODS Version Greater Or Equal Than  1.5.0
+   IF  ${version-check}==True
+      Wait Until Element Is Visible  xpath://div[@class="jsp-spawner__image-options"]
+   ELSE
+      Wait Until Element Is Visible  xpath:/html/body/div[1]/form/div/div/div[2]/div[2]/div[1]
```

Comment:
maybe during refactoring we should change these absolute xpaths

Reply:
We probably won't need the `ELSE` block anymore after 1.6, we need to support N and N-1 only